Unable to start Node-Red after 12.0.1 update, and revert back to 11.1.2

# Problem/Motivation

Node-RED had been running fine until this mornings update to 12.0.1. I noted the breaking changes, but I'm not using them in my flows, so figured it would be fine.

## Expected behavior

NR should display it's UI as expected.

## Actual behavior

Upgrading to 12.0.1 gave a `Bad Gateway` error when opening NR in Ingress. I restored the previous backup I had made, but the issue persists.

Error log below. At this point, I've removed the startup packages from my configuration, and have just a bare bones config, with no optional parameters.

```
[s6-init] making user provided files available at /var/run/s6/etc...exited 0.
[s6-init] ensuring user provided files have correct perms...exited 0.
[fix-attrs.d] applying ownership & permissions fixes...
[fix-attrs.d] done.
[cont-init.d] executing container initialization scripts...
[cont-init.d] 00-banner.sh: executing... 
-----------------------------------------------------------
 Add-on: Node-RED
 Flow-based programming for the Internet of Things
-----------------------------------------------------------
 Add-on version: 11.1.2
 There is an update available for this add-on!
 Latest add-on version: 12.0.1
 Please consider upgrading as soon as possible.
 System: Home Assistant OS 8.1  (amd64 / generic-x86-64)
 Home Assistant Core: 2022.6.4
 Home Assistant Supervisor: 2022.05.3
-----------------------------------------------------------
 Please, share the above information when looking for help
 or support in, e.g., GitHub, forums or the Discord chat.
-----------------------------------------------------------
[cont-init.d] 00-banner.sh: exited 0.
[cont-init.d] 01-log-level.sh: executing... 
[cont-init.d] 01-log-level.sh: exited 0.
[cont-init.d] customizations.sh: executing... 
[cont-init.d] customizations.sh: exited 0.
[cont-init.d] nginx.sh: executing... 
[cont-init.d] nginx.sh: exited 0.
[cont-init.d] node-red.sh: executing... 
patching file nodes/ui_base.html
Hunk #1 succeeded at 1164 (offset 633 lines).
npm ERR! Cannot convert undefined or null to object
npm ERR! A complete log of this run can be found in:
npm ERR!     /root/.npm/_logs/2022-06-08T14_43_16_057Z-debug.log
[10:43:16] FATAL: Failed un-installing conflicting packages
[cont-init.d] node-red.sh: exited 1.
[cont-finish.d] executing container finish scripts...
[cont-finish.d] 99-message.sh: executing... 
-----------------------------------------------------------
                Oops! Something went wrong.

 We are so sorry, but something went terribly wrong when
 starting or running this add-on.
 
 Be sure to check the log above, line by line, for hints.
-----------------------------------------------------------
[cont-finish.d] 99-message.sh: exited 0.
[cont-finish.d] done.
[s6-finish] waiting for services.
[s6-finish] sending all processes the TERM signal.
[s6-finish] sending all processes the KILL signal and exiting.
```

> npm ERR! Cannot convert undefined or null to object

That seems the error NR provides, there is not much I can do with that.

As you have run the latest release already, part of your configuration has probably been migrated. If you restore an NR backup, also restore the configuration (which is in the Home Assistant configuration directory, thus not part of the NR add-on backup).

--- a/node-red/rootfs/etc/cont-init.d/node-red.sh
+++ b/node-red/rootfs/etc/cont-init.d/node-red.sh
@@ -67,5 +67,5 @@ if bashio::fs.file_exists "/config/node-red/package.json"; then
         node-red-contrib-home-assistant \
         node-red-contrib-home-assistant-llat \
         node-red-contrib-home-assistant-ws \
-            || bashio::exit.nok "Failed un-installing conflicting packages"
+            || bashio::log.warning "Failed un-installing conflicting packages"
 fi

